SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.iOS/Crashes.cs

```diff
@@ -174,6 +173,11 @@ static string AnonymizePath(string path)
             return Regex.Replace(path, pattern, "/Users/USER/");
         }
 
+        static NSDictionary StringDictToNSDict(IDictionary<string, string> dict)
+        {
+            return NSDictionary.FromObjectsAndKeys(dict.Values.ToArray(), dict.Keys.ToArray());
+        }
+
```

Comment:
Let me know if you prefer this somewhere else.

Reply:
Can you check what we do on Analytics.TrackEvent? its a 1 liner so I am not too worried about duplication just make sure it's managed the exact same way.